docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java

```diff
@@ -47,45 +45,47 @@
     private CubeContainer videoConverterContainer;
     private String browser;
     private Path videoRecordingFolder;
-    
+
     private final String seleniumContainerName;
     private final String vncContainerName;
     private final String conversionContainerName;
-    private final int seleniumBoundedPort;
-    private final String dockerRegistry;
 
     private SeleniumContainers(String browser, CubeDroneConfiguration cubeDroneConfiguration) {
         this.browser = browser;
-        
-        switch(cubeDroneConfiguration.getContainerNameStrategy()) {
+        final int seleniumBoundedPort;
+
+        final String browserBaseName = Optional.ofNullable(System.getenv("SELENIUM_CONTAINER_BASE_NAME"))
```

Comment:
Nice to add the project name to the broswer image on CI - docker container ls -a